packages/terra-menu/tests/nightwatch/menu-item-group/menu-item-group-spec.js

```diff
@@ -0,0 +1,30 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+
```

Comment:
adding `// eslint-disable-next-line import/no-extraneous-dependencies` to you spec files will remove the following error seen in Travis log;

>  3:20  error  'terra-toolkit' should be listed in the project's dependencies. Run 'npm i -S terra-toolkit' to add it  import/no-extraneous-dependencies